src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -245,51 +270,57 @@
     }
 
     .datagrid-header {
-        background-color: $clr-thead-bgcolor;
         position: sticky;
         top: 0;
-        display: flex;
-        flex: 0 0 auto;
-        border-bottom: $clr-table-borderstyle;
-        border-bottom-width: 2px;
+        // bug(popover): prevents action-overflow from being on top (first row). 
+        // Needed to keep select/radio and expand svgs underneath header on scrolling
         z-index: map-get($clr-layers, datagrid-header);
+        min-height: 1.5rem;
+        width: auto;
 
         .datagrid-row {
+
+            .datagrid-row-sticky {
+                background-color: $clr-thead-bgcolor;
+                z-index: map-get($clr-layers, datagrid-sticky-header);
+            }
             &:hover {
                 background-color: $clr-thead-bgcolor;
+
+                .datagrid-row-sticky {
+                    background-color: $clr-thead-bgcolor;
+                }
             }
         }
+
+        .datagrid-row-scrollable {
+            flex-direction: row;
+        }
     }
 
     .datagrid-table {
+        //flex: 1 1;
```

Comment:
Remove all the commented out styles if they aren't needed

Reply:
done.